futaba/sql/models/settings_model.py

```diff
@@ -1,5 +1,5 @@
 #
-# sql/settings.py
+# sql/models/settings_model.py
```

Comment:
We were considering two routes, either `models/settings` or `settings_model`. I don't think we need both.

Reply:
Ok, I'll change it to /models/<file>.py as that looks better